--- origsrc/LibRaw-0.18.2/Makefile.am	2017-03-04 12:33:42.000000000 -0600
+++ src/LibRaw-0.18.2/Makefile.am	2017-05-05 11:56:20.424977600 -0500
@@ -40,6 +40,8 @@ lib_libraw_r_a_CFLAGS = -pthread -w
 lib_libraw_la_SOURCES = $(lib_libraw_a_SOURCES)
 lib_libraw_r_la_SOURCES = $(lib_libraw_a_SOURCES)
 
+lib_libraw_la_CXXFLAGS = -DLIBRAW_BUILDLIB
+lib_libraw_r_la_CXXFLAGS = -DLIBRAW_BUILDLIB
 lib_libraw_la_LDFLAGS = -no-undefined -version-info $(LIBRAW_SHLIB_VER)
 lib_libraw_r_la_LDFLAGS = -no-undefined -version-info $(LIBRAW_SHLIB_VER)
 
--- origsrc/LibRaw-0.18.2/libraw/libraw_types.h	2017-03-04 12:33:42.000000000 -0600
+++ src/LibRaw-0.18.2/libraw/libraw_types.h	2017-05-05 12:00:37.366543200 -0500
@@ -97,7 +97,7 @@ typedef unsigned char uchar;
 typedef unsigned short ushort;
 
 #ifdef WIN32
-#ifdef LIBRAW_NODLL
+#if defined(LIBRAW_NODLL) || (defined(LIBRAW_BUILDLIB) && !defined(PIC))
 # define DllDef
 #else
 # ifdef LIBRAW_BUILDLIB
